module/VuFind/src/VuFind/Search/Factory/EITBackendFactory.php

```diff
@@ -51,7 +51,7 @@ class EITBackendFactory extends AbstractBackendFactory
     /**
      * Logger.
      *
-     * @var \Laminas\Log\LoggerInterface
+     * @var \VuFind\Log\LoggerAwareInterface
```

Comment:
I think this might be the wrong interface. Possible correction:

```suggestion
     * @var \Psr\Log\LoggerInterface
```